IEEE Xplore.js

```diff
@@ -356,6 +361,10 @@ var testCases = [
 				"volume": "52",
 				"attachments": [
 					{
+						"title": "IEEE Xplore Full Text PDF",
+						"mimeType": "application/pdf"
+					},
+					{
```

Comment:
How did this end up in the tests? Was the test run with user agent set to mobile? Was this not working for desktops also?

If the former, then you should undo this change, because the tests will fail on Zotero test servers, which do not run as mobile. If the latter, then... are PDFs no longer displayed in frames anywhere? (doesn't seem to be consistent with PR description)